citest/json_predicate/binary_predicate.py

```diff
@@ -428,8 +458,19 @@ def __call__(self, value):
     '==', lambda a, b: a == b, operand_type=list)
 LIST_NE = StandardBinaryPredicateFactory(
     '!=', lambda a, b: a != b, operand_type=list)
+
+def lists_equivalent(a, b):
```

Comment:
Good point. I didnt consider that. This way is more correct. Probably
should have been a bag before because multiple "identical" values suggests
something different is going on.

On Thu, Apr 7, 2016 at 10:50 AM, Lars Wander <notifications@github.com>
wrote:

> In citest/json_predicate/binary_predicate.py
> <https://github.com/google/citest/pull/137#discussion_r58884524>:
>
> > @@ -428,8 +458,19 @@ def __call__(self, value):
> >      '==', lambda a, b: a == b, operand_type=list)
> >  LIST_NE = StandardBinaryPredicateFactory(
> >      '!=', lambda a, b: a != b, operand_type=list)
> > +
> > +def lists_equivalent(a, b):
>
> That makes sense. I meant set([1, 2, 2]) == set([1, 2]), while !lists_equivalent([1,
> 2, 2], [1, 2]).
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/google/citest/pull/137/files/a82ce9aa13cfe030abd936cc0de1ee79176f2ab2#r58884524>
>